Newsletters: add 249 (2023-05-03)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Bitcoin Core 25158 @bitschmidty 
- [x] Bitcoin Core 26933 @glozow 
- [x] Bitcoin Core 25325 @Xekyo

bitschmidty commented on _posts/en/newsletters/2023-05-03-newsletter.md:
```diff
@@ -0,0 +1,181 @@
+- **Analysis of signature adaptor security:** Adam Gibson
+  [posted][gibson adaptors] to the Bitcoin-Dev mailing list an analysis
+  of the security of [signature adaptors][topic adaptor signatures],
+  especially about how they interact with [multisignatures][topic
```
Maybe

```suggestion
  especially about how they interact with [multisignature][topic
```

bitschmidty commented on _posts/en/newsletters/2023-05-03-newsletter.md:
```diff
@@ -0,0 +1,181 @@
+- [Bitcoin Core #25325][] introduces a pool based memory resource for
+  the UTXO cache. The new data structure pre-allocates and manages a
+  larger pool of memory to track UTXOs instead of allocating and freeing
+  memory for each UTXO individually. UTXO lookups cause a major part of
```
Maybe

```suggestion
  memory for each UTXO individually. UTXO lookups represent a major proportion of
```

bitschmidty commented on _posts/en/newsletters/2023-05-03-newsletter.md:
```diff
@@ -0,0 +1,181 @@
+  However, after `utxoupdatepsbt` was added to Bitcoin Core, several
+  hardware signing devices began requiring even spends of segwit v0
+  outputs to include full transactions in order to prevent a variation
+  on fee overpayment that could result from a user seemingly signing the
```
```suggestion
  of fee overpayment that could result from a user seemingly signing the
```

harding commented:
Made all edits for @bitschmidty feedback (thanks!), reviewed additions by @bitschmidty, @glozow, @Xekyo (thanks!), added lede, updated releases/RCs, and add topic page entries.  Thanks everyone!

bitschmidty commented on _topics/en/psbt.md:
```diff
@@ -195,6 +195,9 @@ optech_mentions:
   - title: "LND #7122 adds support for importing PSBTs from binary files"
     url: /en/newsletters/2022/11/30/#lnd-7122
 
+  - title: "Bitcoin Core #25939 allows nodes with txindex enabled to add txes with `utxoupdatepsbt` RPC"
```
```suggestion
  - title: "Bitcoin Core #25939 allows nodes with txindex enabled to add txs with `utxoupdatepsbt` RPC"
```

bitschmidty commented:
One small fixup in the topic review. Squashed and merged! Thanks @harding @Xekyo @glozow for contributing this week!
